test/old/testdir/setup.vim

```diff
@@ -17,7 +17,7 @@ if exists('s:did_load')
   set nohidden nosmarttab noautoindent noautoread noruler noshowcmd
   set nohlsearch noincsearch
   set nrformats=bin,octal,hex
-  set shortmess=filnxtToOS
+  set shortmess=ilnxtToOS
```

Comment:
No

Reply:
Care to elaborate (preferably with more than one syllable)? It does not cause any tests to fail.

Reply:
As a check that having "f" in 'shortmess' doesn't cause an error

Reply:
Makes sense. Reverted.